packages/fast-components-react-base/src/image/image.tsx

```diff
@@ -47,8 +47,9 @@ class Image extends Foundation<ImageHandledProps, ImageUnhandledProps, {}> {
         } else {
             className = `${className} ${get(
                 this.props,
-                "managedClasses.image__picture"
-            )}`;
+                "managedClasses.image__picture",
+                ""
+            )}`.trim();
```

Comment:
As noted by @nicholasrice and @chrisdholt you can remove all `trim()` from strings going into `super.generatClassNames` because it will trim these strings for you

Reply:
I see now that we are doing trim in foundation class. Thanks!